### ISSUE TITLE
Four core bio sources are throwing compile exceptions in current gradle branch

### ISSUE THREAD

**Issue by sammyjava** [intermine/intermine#1837]
FYI. Presumably due to model changes, these four bio sources have to be commented out to be able to install bio/sources. The exception is on organism.getTaxonId().
```
//project(':bio-source-chado-db').projectDir = new File(settingsDir, './chado-db')
//project(':bio-source-fasta').projectDir = new File(settingsDir, './fasta')
//project(':bio-source-panther').projectDir = new File(settingsDir, './panther')
//project(':bio-source-pubmed-gene').projectDir = new File(settingsDir, './pubmed-gene')
```

**Comment by julie-sullivan** [comment]
Sorry! I messed up the branches, I'll try to revert.

**Comment by sammyjava** [comment]
No biggy, just keep moving forward if that's the direction it's headed. There should be an "FYI" flag for issues.

**Comment by julie-sullivan** [comment]
Ha yes! :) FYI I merged the `gradle` + `dev` + `model-change` branch all into one ==> `gradle-merge` in my repo. I've merged in all the pull requests and this will be the InterMine 2.0 release candidate. There won't be any more features or changes, just bug fixes.

FlyMine and HumanMine (beta) are on this new `gradle-merge` branch and everything looks good so far. Still testing, then I'm going to push this branch to the InterMine repo as the `release-candidate` branch.

I didn't mean to update the branch you are using! Sorry. Merging all three branches was a nightmare and I may have been mad at github. Which branch is that btw? Looking at my gradle branch, it looks like Organism.taxonId is still an Integer. I've changed it to be a String and I thought this was the error.

**Comment by sammyjava** [comment]
It's still an Integer in the current intermine:gradle (bio/model/core.xml at least).

**Comment by julie-sullivan** [comment]
Is that the branch you are using? I'll revert everything I did yesterday!

**Comment by sammyjava** [comment]
I can use any branch you tell me to use. I thought that's the one I'm supposed to use, the current "official" gradle branch of intermine.

**Comment by sammyjava** [comment]
I'm happy to alpha the Organism.taxonId -> String stuff as well. There are a lot of places, which I've already hit since I'd gone and done most the changes not knowing that it was still Integer.

**Comment by julie-sullivan** [comment]
Well I was hoping to get you working on the gradle branch, then fix the things to get over to the release-candidate branch. :( But apparently I've mucked that up.

You have the patience of a saint!! I couldn't do this without you!

I've just pushed a `release-candidate` branch to the InterMine repo. Try that one? 

1. Has all the model changes. 
2. Has updated biojava in bio-webapp and fasta. I don't think this should interfere with your biojava but just be aware.

This is all of InterMine 2.0! Thanks!!
